CMakeLists.txt

```diff
@@ -310,31 +332,29 @@ openms_export_targets()
 install_export_targets()
 
 #------------------------------------------------------------------------------
-# build settings and configs for external code
+# Installation and packaging:
 #------------------------------------------------------------------------------
-# install routines for ..
+
+
+# packaging routines:
 if(NOT "${PACKAGE_TYPE}" STREQUAL "none")
-  ## Needed in package_general.cmake
-  set(PACKAGE_LIB_DIR lib)
-  set(PACKAGE_BIN_DIR bin)
-  
+  include(cmake/package_general.cmake)
   if("${PACKAGE_TYPE}" STREQUAL "dmg") # .. MacOSX
-    set(PACKAGE_BIN_DIR TOPP)
-    include(cmake/package_general.cmake)
     include(cmake/package_dragndrop_dmg.cmake)
   elseif("${PACKAGE_TYPE}" STREQUAL "rpm") # .. rpms
-    include(cmake/package_general.cmake)
     include(cmake/package_rpm.cmake)
   elseif("${PACKAGE_TYPE}" STREQUAL "deb") # .. debs
-    include(cmake/package_general.cmake)
     include(cmake/package_deb.cmake)
-    elseif("${PACKAGE_TYPE}" STREQUAL "nsis") # .. Windows nsis
-    set(PACKAGE_LIB_DIR bin)
-    include(cmake/package_general.cmake)
+  elseif("${PACKAGE_TYPE}" STREQUAL "nsis") # .. Windows nsis
```

Comment:
in the future we could also consider using wix instead of NSIS. Seems to be cleaner.